Request to add a support issues template to the repository

Vaerian opened the pull request:
The majority of issues made on the fabric repository can be classified as either a legitimate issue relating to fabric api and its development, or a user troubleshooting issue. Obviously the former is the intended use of the issues section, and the latter results in a lot of unnecessary clutter in the issues section, and while I recognize that no solution will completely prevent misuse of the issues section, adding some issues templates might reduce the amount of invalid issues, or at the very least help manage them better.

I propose first that an issues label be added called "support" in order to distinguish between a development issue and a support issue. This PR adds an issue template for support issues that automatically adds that support tag.

The default contents of the issue template contains information for users on how to troubleshoot, and where to get actual support (on the fabric discord). Worst case scenario they ignore all of that and we have an extra issue, but this time it's actually sorted into a category. Best case scenario the issues section isn't unnecessarily cluttered, somebody actually got the help they needed, and everyone is happy.

The goal is to eventually make some more PRs like this to add helpful issue templates for new module ideas, etc... but this one seemed specifically helpful.

Commit:
Added a support issues template

TheBrokenRail commented on .github/ISSUE_TEMPLATE/support.md:
```diff
@@ -0,0 +1,23 @@
+title: "[Support] [TITLE OF YOUR ISSUE]"
+labels: support
+---
+The issues of the fabric GitHub repository is not the best place for you to get support for your issue. If you would like better and more immediate help with your issue please join the fabric discord server [here](https://discord.gg/v6v4pMv), and ask for help in the #player-support channel.
+
+If you are not getting the help you need on the discord server then try the following steps to resolve the issue yourself.
+
+1. Check that your fabric loader installation is correct by removing all of the mods from your mod folder and running the game. If the game successfully loads continue to the next step. If not, then read the instructions [here](https://fabricmc.net/wiki/install) on how to properly install the fabric loader.
+2. Remove all of the mods from your mod folder, and add them back one by one until you find which mod is the culprit. When you find which mod is responsible for the crashing, then look for a link to the sources for the mod and make an issue on the repository for that mod.
+3. If none of that has worked then delete everything above this line (including this line) in the issue template, and fill in the rest of the template with information about your specific issue. Each of the fields are a title wrapped in two asterisks on either side. Next to them is a descriptor written in all caps wrapped in square brackets on either side. Replace the descriptor and the square br ...
```
As Fabric is a name, shouldn't it be capitalized?

Vaerian replied:
Should be resolved in https://github.com/FabricMC/fabric/pull/1051/commits/e4db6628a84f38b0e2acf8116790ea7088bdb2ed

i509VCB commented on .github/ISSUE_TEMPLATE/support.md:
```diff
@@ -0,0 +1,23 @@
+If you are not getting the help you need on the discord server then try the following steps to resolve the issue yourself.
+
+1. Check that your fabric loader installation is correct by removing all of the mods from your mod folder and running the game. If the game successfully loads continue to the next step. If not, then read the instructions [here](https://fabricmc.net/wiki/install) on how to properly install the fabric loader.
+2. Remove all of the mods from your mod folder, and add them back one by one until you find which mod is the culprit. When you find which mod is responsible for the crashing, then look for a link to the sources for the mod and make an issue on the repository for that mod.
```
I'd probably throw a space between each step so it doesn't look so wall of texty

Vaerian replied:
Should be resolved in https://github.com/FabricMC/fabric/pull/1051/commits/e4db6628a84f38b0e2acf8116790ea7088bdb2ed

Commit:
Minor tweaks based on suggestions
- Added spaces between the list items so it doesn't look like a giant wall of text
- Capitalized "Fabric" as it is technically a proper noun
- Rephrased some of the verbage in the template so it's easier to understand
- Used a horizontal rule to clarify how to proceed if none of the above troubleshooting steps have solved the problem

haykam821 commented:
This won't be needed if/when GitHub releases their Discussions feature.
